.github/workflows/build_docs.yml

```
@@ -0,0 +1,43 @@
+name: tests
+
+on:
+  push:
+    branches:
+      - dev
+      - main
+      - releasing/*
+  pull_request:
+    head_ref-ignore:
```

actionlint (1.7.12)

[error] unexpected key "head_ref-ignore" for "pull_request" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)